types/interfaces/interface.ts

```diff
@@ -0,0 +1,67 @@
+export interface VideoItem {
+  snippet: {
+    title: string
+    publishedAt: string
+    channelTitle: string
+    resourceId: {
+      videoId: string
+    }
+    tags: string[]
+    thumbnails: {
+      default: {
+        url: string
+        width: number
+        height: number
+      }
+      medium: {
+        url: string
+        width: number
+        height: number
+      }
+      high: {
+        url: string
+        width: number
+        height: number
+      }
+      standard: {
+        url: string
+        width: number
+        height: number
+      }
+    }
+  }
+}
```

Comment:
weird naming :D `/interfaces/interface.ts ` would be better to split these two into separate files and call them `VideoItem.ts` and `PodcastEpisode.ts`....but this is just a tip for a better code organization, you can keep it like this if you want 👍

Reply:
When I Iooked around, because I was debating initially as well should it go into separate files or not, it was 50/50 on what to do. I went with the suggestion to keep it in one file as it said that that may be better (easier) for small projects, but I'l put it into two separate ones now that I got your opinion on it :)